test/user.test.js

```diff
@@ -0,0 +1,63 @@
+const { connectDatabase, disconnectDatabase } = require("../utils/test-utils/db.util");
+
+const supertest = require('supertest')
+const app = require('../app');
+const request = supertest(app);
+
+const User = require('../db/schema/user');
+const userTestData = require('./user.test.data');
+
+describe('user routes', () =>{
```

Comment:
What do we expect if the input is invalid for GET and POST? What about if the client is trying to get a nonexistent user?

Reply:
In terms of GET you wouldn't really get an invalid since there isn't any input. 
As for POST I'll update tests to accommodate for that 👍